Algorithm.CSharp/CoarseFundamentalTop5Algorithm.cs

```diff
@@ -66,7 +66,7 @@ public static IEnumerable<Symbol> CoarseSelectionFunction(IEnumerable<CoarseFund
         //Data Event Handler: New data arrives here. "TradeBars" type is a dictionary of strings so you can access it by symbol.
         public void OnData(TradeBars data)
         {
-            Console.WriteLine($"OnData({UtcTime:o}): Keys: {string.Join(", ", data.Keys.OrderBy(x => x))}");
+            Log($"OnData({UtcTime:o}): Keys: {string.Join(", ", data.Keys.OrderBy(x => x))}");
```

Comment:
Ironically, these were purposefully `Console.WriteLine` statements. Using `Console.WriteLine` from within the regression tests pipes it directly to `Log.Trace` so they're written synchronously with other logging statements. If we use `Log`, then those messages get queued and then dequeued in the result handler a few seconds later, making debugging some data feed issues much harder.

*Also*, since `Console.WriteLine` pipes to `algorithm.Log` in normal circumstances (running locally or in QC cloud), I think it's fine to leave as `Console.WriteLine`.

Another also, the `forward-console-messages` configuration value is what controls this piping behavior. `true` forwards the messages to `algorithm.Log` while `false` sends them directly to the screen (IMO, as you would expect from a `Console.WriteLine` statement)